Jenkinsfile

```
@@ -49,3 +55,13 @@ def runPublishingApiPactTests(govuk) {
     }
   }
 }
+def runCollectionsOrganisationApiPactTests(govuk){
```

```suggestion
def runCollectionsPactTests(govuk) {
```

I realise this is true at the moment, but I guess it helps to keep it generic for if we were to add pact tests for other parts of Collections.